data/tmc-devices.json

```diff
@@ -566,7 +572,7 @@
                             }
                         },
                         "properties": {
-                            "SdpSubarrayFQDN": [
+                            "SdpSubarrayNodeFQDN": [
```

Comment:
Property name should be SdpSubarrayFQDN and not SdpSubarrayNodeFQDN.

Reply:
In general, the term 'node' implies the physical device representing the subarray. In our case, a tango device representing the subarray can be called a subarray node, whereas if you refer to just 'subarray', it would imply all the devices and resources that participate in the subarray. The term used depends on the context and is used interchangeably.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd